### ISSUE TITLE
List items on /plugins need a parent ul or ol

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75598]
### Quick summary

Thirty list items on /plugins have a div instead of a ul or ol parent. This can cause a problem for people who use accessibility assisting devices like screen readers as the screen reader uses the ul tags to tell where the list starts and ends.

### Steps to reproduce

1. On /plugins
2. Inspect any list item on the page or run an accessibility checker like the axe DevTools Chrome extension.

### What you expected to happen

All list items should have an ul or ol parent.

### What actually happened

There are thirty accessibility rule violations due to missing parents.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable
